setup.py

```diff
@@ -34,7 +34,7 @@
         'sqlalchemy',
         'click',
         'datapackage',
-        'jtssql',
+        'jsontableschema',
```

Comment:
I think we'll also have to wait for all constraints to be implemented as well.

Just wanted to make sure I'm using the API correctly.